sleap/gui/commands.py

```diff
@@ -2709,6 +2731,27 @@ def do_action(context: CommandContext, params: dict):
 
         context.execute(SetSelectedInstanceTrack, new_track=new_track)
 
+class SetSelectedInstanceGroup(EditCommand):
+    @staticmethod
+    def do_action(context, params):
+        selected_instance = context.state["selected_instance"]
```

Comment:
The correct key is `"instance"`
```suggestion
        selected_instance = context.state["instance"]
```